lk2nd/device/dts/msm8909/rules.mk

```diff
@@ -6,3 +6,6 @@ QCDTBS += \
 	$(LOCAL_DIR)/msm8905-qrd-skub.dtb \
 	$(LOCAL_DIR)/msm8909-1gb-qrd-skuc.dtb \
 	$(LOCAL_DIR)/msm8909-mtp.dtb \
+
+ADTBS += \
```

Comment:
Appended DTB? Doesn't it work as QCDTB?

Reply:
No: Stock fastboot gives "dtb not found", unless you want it in both?